.github/workflows/upload_package.yml

```diff
@@ -0,0 +1,62 @@
+name: Release Package
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine jupyter urllib3 pandas pyyaml
+        python setup.py sdist bdist_wheel
+    - name: Publish distribution 📦 to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        password: ${{ secrets.pypi_password }}
+    - name: Run Changelog
+      run: |
+        jupyter nbconvert --to notebook --execute --inplace --ExecutePreprocessor.timeout=-1 --ExecutePreprocessor.kernel_name=python3 tools/gitcount.ipynb
+    - name: Cat Changelog
+      uses: pCYSl5EDgo/cat@master
+      id: changetxt
+      with:
+        path: ./tools/changelog.md
+      env:
+        TEXT: ${{ steps.changetxt.outputs.text }}
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+      with:
```

Comment:
update workflow based on [that of `libpysal`](https://github.com/pysal/libpysal/blob/b1ddcd7b769bf85a297cfe965f4aab9672d95415/.github/workflows/release_and_publish.yml#L49-L59), which is updated and streamlined